contrib/clojure-package/README.md

```diff
@@ -105,9 +105,11 @@ brew install opencv
 - Create a new project with `lein new my-mxnet`
 - Edit your `project.clj` and add one of the following entries to `:dependencies`, based on your system and the compute device you want to use:
 
-  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-cpu "1.5.0"]`
-  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-gpu "1.5.0"]`
-  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-osx-cpu "1.5.0"]`
+  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-cpu "latest-version"]`
+  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-gpu "latest-version"]`
+  - `[org.apache.mxnet.contrib.clojure/clojure-mxnet-osx-cpu "latest-version]`
```

Comment:
Does this work literally or is it just a placeholder? For the latter case, maybe do `"<latest-version>"` or something that makes clear that copy-paste won't work.

Reply:
It's a placeholder - good call on the brackets - will fix 👍